docs/service_specs/bff.md

```diff
@@ -0,0 +1,30 @@
+﻿[← Back Home](../README.md)
+
+> ⚠️ **Note:** This service is planned to be renamed to "admin-bff".
+
+## Backend for Frontend (BFF) Overview
+
+- **Technology**: .NET Core
+- **Image Name**: link-bff
+- **Port**: 8080
+- **Database**: NONE
+- **Scale**: 0-3
+
+## Environment Variables
+
+| Name                                        | Value                         | Secret? |
+|---------------------------------------------|-------------------------------|---------|
+| Link__Audit__ExternalConfigurationSource    | AzureAppConfiguration         | No      |
+| ConnectionStrings__AzureAppConfiguration    | `<AzureAppConfigEndpoint>`    | Yes     |
+| GatewayConfig:KafkaBootstrapServers:0       | `<KafkaBootstrapServer>`      | No      |
+| GatewayConfig:AuditServiceApiUrl            | `<URL> (without /api)`        | No      |
+| GatewayConfig:NotificationServiceApiUrl     | `<URL> (without /api)`        | No      |
+| GatewayConfig:TenantServiceApiUrl           | `<URL> (without /api)`        | No      |
+| GatewayConfig:CensusServiceApiUrl           | `<URL> (without /api)`        | No      |
+| GatewayConfig:ReportServiceApiUrl           | `<URL> (without /api)`        | No      |
+| GatewayConfig:MeasureServiceApiUrl          | `<URL> (without /api)`        | No      |
+| IdentityProviderConfig:Issuer               | ??                            | No      |
+| IdentityProviderConfig:Audience             | ??                            | No      |
+| IdentityProviderConfig:NameClaimType        | email                         | No      |
+| IdentityProviderConfig:RoleClaimType        | roles                         | No      |
+| IdentityProviderConfig:ValidTypes           | `[ "at+jwt", "JWT" ]`         | No      |
```

Comment:
I'm going to wait to address #2, 3 and 4 until a future round of documentation updates so that this PR can stay focused on just migrating the existing documentation from another system